packages/javascript/FlatfileJavascript.ts

```diff
@@ -271,13 +247,109 @@ function initializeIFrameConfirmationModal(
   }
 }
 
+type InitSpaceType = ISpace & {
+  isAutoConfig: boolean
+}
+
+// TODO: Replace hardcoded type with imported type from Platform
+export interface InitialResourceData {
```

Comment:
Is the latest `@flatfile/api` released with these types?

Reply:
No - since it is just a BFF endpoint - neither the endpoint nor the types are exposed via an package